cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html

```diff
@@ -37,8 +37,8 @@
                 'image': {
                       'upload': {
                                 'url': 'http://placehold.it/300x300',
-                                'alt': ''
-                      }
+                      },
+                      'alt': ''
```

Comment:
@sebworks if you want. i think whoever put that there was trying to show case the image as a bg